src/modules/10-common/components/UserGroupsInput/FormMultitypeUserGroupInput.tsx

```diff
@@ -26,7 +26,7 @@ export interface FormMultiTypeUserGroupInputProps
   extends Omit<ExpressionAndRuntimeTypeProps, 'fixedTypeComponent' | 'fixedTypeComponentProps'> {
   label: string
   tooltipProps?: DataTooltipInterface
-  formik?: FormikContext<any>
+  formik?: FormikContextType<any>
```

Comment:
Codacy has a fix for the issue: [Unexpected any. Specify a different type.](https://app.codacy.com/gh/harness/harness-core-ui/pullRequest?prid=9232251)
```suggestion
  formik?: FormikContextType<unknown>
```